tests/unittests/test_endpoint_logic.py

```diff
@@ -293,7 +293,7 @@ def get_group_references(group):
     processed_data = json.loads(json.dumps(paper_network.get_papernetwork(input_js_paper_network["response"]["docs"], 10), sort_keys=True))
 
 
-    self.assertEqual(processed_data, test_js_paper_network)
+    #self.assertEqual(processed_data, test_js_paper_network)
```

Comment:
This uncommented line was merged, it should be reviewed and ascertained why the test was not passing